src/styles/index.scss

```diff
@@ -1,3 +1,86 @@
+$stopwatch-size: 80vmin;
+$center-size: 5vmin;
+$minutes-hand-hight: 20vmin + $center-size / 2;
```

Comment:
There is a typo in the variable name `$minutes-hand-hight`. It should be `$minutes-hand-height` to correctly spell 'height'. This typo is repeated in `$seconds-hand-hight` on line 5.